stage1/00-init.sh

```diff
@@ -13,3 +13,7 @@ pacman-key --populate archlinuxarm
 
 info "Updating system"
 pacman -Syu --noconfirm
+
+info "Setting up user accounts"
+useradd -mp $(openssl passwd -crypt tobor) robot
```

Comment:
Is openssl a dep here? Can we use chpasswd instead? Maybe even just a static hash?